Game/multiplayer-project/Assets/Scripts/Http/Models/query_data.json

```diff
@@ -0,0 +1,21 @@
+{
+    "data":{
+        "player_id":0,
+        "bot_id":"0",
+        "query":"what time is it"
+    },
+    "POI":{
+        "harvest":{
```

Comment:
let's try to have always the same keys in dict, action should be in value dict with "action" key for example

...
"action" : {
   "action" : ["harvest"], # migh be several actions | doesn't care there is 2 'actions' keys consecutives
   ...